.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "workshops/eventmesh-teq-kafka/okafka"]
```

Comment:
why this?

Reply:
okafka project is being loaded as submodule (git submodule). I remove the code copied. It is a best practice when we need another project.